gateway/1.4/modules/ROOT/pages/flex-conn-manage-public-api.adoc

```diff
@@ -1,18 +1,18 @@
-= Manage Flex Gateway Using the API Manager API
+= Configuring Flex Gateway Using the API Manager API
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images
 :page-mode: conn
 
-You can manage Flex Gateway in Connected Mode using the API Manager API from the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/[Anypoint Platform Developer Portal^] instead of using the API Manager user interface through Anypoint Platform. This enables you to use the command line to communicate with API Manager. Using the steps in this tutorial you can create, deploy, apply a policy to, and delete an API instance.
+You can configure Flex Gateway in Connected Mode using the API Manager API from the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/[Anypoint Platform Developer Portal^] instead of using the *API Manager* user interface through Anypoint Platform. This enables you to use the command line to communicate with *API Manager*. Using the steps in this tutorial you can create, deploy, apply a policy to, and delete an API instance.
```

Comment:
```suggestion
You can configure Flex Gateway in Connected Mode using the API Manager API from the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/[Anypoint Platform Developer Portal^] instead of using the API Manager user interface through Anypoint Platform. This enables you to use the command line to communicate with API Manager. Use the steps in this tutorial to create, deploy, apply a policy to, and delete an API instance.
```

Reply:
Avoid using "this" alone as the subject. This process? This tool? This method?

https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22it,%22%22this,%22%22these%22

Reply:
Fixed